<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarWidget</class>
 <widget class="QWidget" name="CarWidget">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="markLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>420</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>380</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Закрыть</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>50</y>
     <width>301</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string>Описание автомобиля</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Марка:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="markLabel_2">
      <property name="text">
       <string>mark</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Модель: </string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="modelLabel">
      <property name="text">
       <string>model</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Стоимость / День:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="costLabel">
      <property name="text">
       <string>cost</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Об автомобиле:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QLabel" name="aboutLabel">
      <property name="text">
       <string>about</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="orderBox">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>225</y>
     <width>291</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>Аренда автомобиля</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0" colspan="2">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Конечная дата</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="orderPassDate">
          <property name="maximumDate">
           <date>
            <year>2020</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>2019</year>
            <month>9</month>
            <day>14</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Начальная дата</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="orderDate">
          <property name="maximumDate">
           <date>
            <year>2020</year>
            <month>12</month>
            <day>31</day>
           </date>
          </property>
          <property name="minimumDate">
           <date>
            <year>2019</year>
            <month>9</month>
            <day>14</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>dd.MM.yyyy</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Арендовать</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Стоимость</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="fullCostLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>xvalue</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>420</width>
     <height>340</height>
    </rect>
   </property>
   <property name="title">
    <string>Фотография</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="resourceLabel">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>300</height>
       </size>
      </property>
      <property name="text">
       <string>Photo</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="header">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QToolButton" name="editButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Редактировать</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoRepeat">
     <bool>false</bool>
    </property>
    <property name="autoExclusive">
     <bool>false</bool>
    </property>
    <property name="autoRaise">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QToolButton" name="deleteCarButton">
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
